payjoin-cli/src/main.rs

```diff
@@ -77,6 +78,7 @@ fn cli() -> ArgMatches {
                 .action(clap::ArgAction::SetTrue)
                 .help("Retry the asynchronous payjoin request if it did not yet complete"),
         )
+        .arg(Arg::new("db_path").short('d').long("db-path").help("Sets a custom database path"))
```

Comment:
Nit: we should normalize the use of dash vs. underscore separators in cli arguments. For example, the `pj_endpoint` and `pj_directory` args use underscores. Looks like a dash is used in all other places.

Reply:
dashes should be used